lib/exp.js

```diff
@@ -783,11 +783,17 @@ function getListType (type, returnType, isMulti) {
       break
     case lists.returnType.NONE:
     default:
-      console.log('getListType valueType:' + type + 'returnType:' + returnType + ' FAILED')
+      console.log("getListType valueType:"+type+"returnType:"+returnType+" FAILED")
```

Comment:
Just a stylistic note: JavaScript template literals are the best way to embed expressions into strings now.

E.g. instead of

    "foo: " + foo + ", bar: " + bar

use

    `foo: ${foo}, bar: ${bar}`

Ref: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals